src/OpenBinary/reader.jl

```diff
@@ -242,12 +266,14 @@ function PSRI.open(
     end
     @assert name_length > 0
 
-    _stage_type = if stage_type == 2#PSR_STAGETYPE_MONTHLY
+    _stage_type = if stage_type == 2
         PSRI.STAGE_MONTH
-    elseif stage_type == 1#PSR_STAGETYPE_WEEKLY
+    elseif stage_type == 1
         PSRI.STAGE_WEEK
-    elseif stage_type == 5#PSR_STAGETYPE_DAILY
+    elseif stage_type == 5
         PSRI.STAGE_DAY
+    elseif stage_type == 10
+        PSRI.STAGE_YEAR
     else
```

Comment:
Shouldn't we take this PR as an opportunity to compare with the `PSR_STAGETYPE_...` or maybe create a constant inside the package. `stage_type == 10` seems quite mysterious for other people